Files/Filesystem/NetworkDrivesManager.cs

```diff
@@ -112,7 +112,7 @@ private async Task SyncSideBarItemsUI()
                     MainPage.SideBarItems.BeginBulkOperation();
 
                     var section = MainPage.SideBarItems.FirstOrDefault(x => x.Text == "SidebarNetworkDrives".GetLocalized()) as LocationItem;
-                    if (section == null)
+                    if (section == null && this.drivesList.Count > 0)
```

Comment:
I don't think this will work, not all mapped drives will have a letter.

Reply:
I'm not sure if I get your point about drive letters, any detected network drives should be in that list (+ network folder), so my thought was that as long as there are no drives in that list (apart from the network folder itself) we do not create the section in the sidebar. Or did I get something wrong?